sys-apps/baselayout/baselayout-9999.ebuild

```diff
@@ -9,7 +9,7 @@ CROS_WORKON_REPO="git://github.com"
 if [[ "${PV}" == 9999 ]]; then
 	KEYWORDS="~amd64 ~arm ~arm64 ~x86"
 else
-	CROS_WORKON_COMMIT="abbf6b604659f780846a881a6c6bafbcd8e2da87" # flatcar-master
+	CROS_WORKON_COMMIT="003645f150af7a331307a3a472ae1f7f1014b0ba" # flatcar-master
```

Comment:
just asking - why don't we directly pull the latest commit (`bbe3f2589c46f61a2c9628ab5dd19e31ff651477`) ?

Reply:
I don't want to introduce the other changes that go through a regular Alpha→Beta→Stable transition